src/github.rs

```diff
@@ -0,0 +1,29 @@
+/*!
+GitHub related logic
+
+# Documentation
+- <https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables>
+- <https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions>
+*/
+
+use std::env;
+
+use once_cell::sync::Lazy;
```

Comment:
Could you help me out understanding you choice of Type here?
The docs say `A OnceCell might store arbitrary non-Copy types, can be assigned to at most once and provides direct access to the stored contents.`. Okay? That sounds like a normal non mutable variable to me. For `sync` it adds `A thread-safe cell which can be written to only once.` ah so this would imply `IS_RUN.._ACTION` is called from multiple threads? Further with `Lazy` we have `A value which is initialized on the first access.` ah classic lazy loading. 

So conclusion: `IS_RUN_AS_GITHUB_ACTION()` is called from multiple threads so it must be thread safe. And it must be lazy because it gets often/sometimes called without it contents being accessed, which means being lazy increases performance. However, I do not see it being used in a way that justifies "the overhead" of being lazy. 

What am I misunderstanding?